langchain/src/document_loaders/fs/unstructured.ts

```diff
@@ -15,11 +15,16 @@ interface Element {
 }
 
 export class UnstructuredLoader extends BaseDocumentLoader {
-  constructor(public webPath: string, public filePath: string) {
+  constructor(
+    public webPath: string,
```

Comment:
Unrelated to this PR, but I assume its on purpose you don't want a default value for this?

Reply:
Actually it probably makes the most sense to default this to our hosted web API. I can update that.

Reply:
Alright this is defaulting to the URL for our hosted API now.